templates/react-papi-tailwind/src/App.tsx

```diff
@@ -1,36 +1,45 @@
 import { type ChainId } from "@reactive-dot/core";
-
+import { ChainProvider, ReactiveDotProvider } from "@reactive-dot/react";
 import { ConnectionButton } from "dot-connect/react.js";
+import { Suspense, useState } from "react";
 
 import "./App.css";
-import { ChainSwitch } from "./components/ChainSwitch";
 import { ChainPage } from "./ChainPage";
-
 import polkadotLogo from "./assets/polkadot-logo.svg";
-
 import { AccountList } from "./components/AccountList";
+import { ChainSwitch } from "./components/ChainSwitch";
+import Loading from "./components/Loading";
+import { config } from "./reactive-dot";
 
-function App({ chainId, setChainId }: { chainId: ChainId; setChainId: React.Dispatch<React.SetStateAction<ChainId>> }) {
-  return (
-    <>
-      <div className="fixed right-10 top-10">
-        <div className="ml-5 inline-block">
-          <ConnectionButton />
-        </div>
-        <div className="ml-5 inline-block">
-          <ChainSwitch chainId={chainId} setChainId={setChainId} />
-        </div>
-      </div>
-
-      <img src={polkadotLogo} className="logo mx-auto h-52 p-4" alt="Polkadot logo" />
-      <div className="container mx-auto p-2 leading-6">
-        <ChainPage />
-      </div>
+function App() {
+  const [chainId, setChainId] = useState<ChainId>("paseo");
 
-      <div>
-        <AccountList />
-      </div>
-    </>
+  return (
+    <ReactiveDotProvider config={config}>
+      <Suspense>
```

Comment:
This might look redundant (it won't trigger any visual render), but per the docs, there should always be a root `Suspense` boundary. Cuz any suspense zapping through and re-mounting the root provider might cause unwanted behaviour.